### ISSUE TITLE
Drop Lato in favor of a native font stack

### ISSUE THREAD

**Issue by leofeyer** [contao/contao#98]
You might be aware that [Apple has disabled Subpixel antialiasing for text in macOS Mojave](https://ahmadawais.com/fix-macos-mojave-font-rendering-issue/), so this is how the Contao back end looks in Chrome and Firefox now:

<img width="832" alt="" src="https://user-images.githubusercontent.com/1192057/46086678-e6769e00-c1a8-11e8-998f-a202dff0f1e5.png">

The problem only occurs on Retina displays, where we use extra thin fonts. A short-term fix is to simply drop those thin fonts:

<img width="837" alt="so bad" src="https://user-images.githubusercontent.com/1192057/46086760-1de54a80-c1a9-11e8-936a-9539c7e13930.png">

However, in the long run (Contao 4.7), I would prefer to drop Lato in favor of a native font stack as used by [Bootstrap](https://getbootstrap.com/docs/4.1/content/reboot/#native-font-stack), GitHub, Gitlab, Medium and many more.

```css
/* Text */
font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;

/* Code */
font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
```

Here is how it would look like:

<img width="837" alt="" src="https://user-images.githubusercontent.com/1192057/46086950-8e8c6700-c1a9-11e8-8b73-3b722d6d8cb0.png">

<img width="781" alt="" src="https://user-images.githubusercontent.com/1192057/46086957-92b88480-c1a9-11e8-97d6-6410a8ca7adf.png">

@contao/developers @fritzmg @frontendschlampe What do you think?

**Comment by contaoacademy** [comment]
Wordpress also switched to a native font stack…
`-apple-system, system-ui, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;`

**Comment by ausi** [comment]
👍 for using the system font stack.

But we should be aware that this makes Contao look differently across browsers/devices.

**Comment by leofeyer** [comment]
Adding [`system-ui`](https://caniuse.com/#feat=font-family-system-ui) makes absolute sense. We can drop `BlinkMacSystemFont` entirely then.

> But we should be aware that this makes Contao look differently across browsers/devices.

Yes, but it also makes Contao look a little more like a native app, which is good I guess.

**Comment by bytehead** [comment]
Go for it.

**Comment by Toflar** [comment]
Go for it.

**Comment by contaoacademy** [comment]
Der Contao Manager nutzt ja ebenfalls "Lato". Es wäre ja eigentlich sinnvoll, wenn dann dort ebenfalls der Font-Stack geändert wird.

**Comment by xchs** [comment]
> Der Contao Manager nutzt ja ebenfalls "Lato".

Related tickets: https://github.com/contao/contao-manager/issues/20, https://github.com/contao/contao-manager/issues/85

**Comment by fritzmg** [comment]
Some Screenshots from Win 10 x64, Firefox, 1.5 ddpx with that font stack, which will use Segoe UI:

![screenshot_2018-10-04 home - contao open source cms](https://user-images.githubusercontent.com/4970961/46461359-665ac480-c7b6-11e8-9bb2-0cc0fbf5cbf9.png)

![screenshot_2018-10-04 articles home content elements - contao open source cms](https://user-images.githubusercontent.com/4970961/46461358-665ac480-c7b6-11e8-81bd-e084d91d95dd.png)

![screenshot_2018-10-04 articles home content elements edit content element id 1 - contao open source cms](https://user-images.githubusercontent.com/4970961/46461356-65c22e00-c7b6-11e8-9b10-5ffc189e0c63.png)

Personally I think that some texts and labels which currently have a font-weight of 700 could be reduced to 600 with that font stack. Also the text within TinyMCE looks a bit nicer with Lato imho. Otherwise I don't see any glaring issues.

**Comment by leofeyer** [comment]
> some texts and labels which currently have a font-weight of 700 could be reduced to 600

I think so, too. Same here on Mac

**Comment by leofeyer** [comment]
Implemented in 86f5fefde96b41c06bc82d164ef090acef651a9a.

**Comment by aschempp** [comment]
Also implemented in https://github.com/contao/contao-manager/commit/70d14c7bf57e25352ad47a3d53979afb159b5a7c
